ubuntu-18-base/generic-setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -e
+set -x
+
+apt-get -qq update
+apt-get -qq install -y --no-install-recommends wget python3-dev python3-pip build-essential sudo git vim dh-autoreconf ninja-build ca-certificates valgrind
```

Comment:
I wonder if we should start adding some more useful packages to some of these docker images, not sure if the base one is the best place to put them.  But packages like openmpi..

Reply:
https://github.com/Alpine-DAV/ascent/blob/4dcab4aa30c0e4fd23b6fa03ef5f00d3db16833a/src/examples/docker/ubuntu/Dockerfile#L49

Reply:
includes many common deps in case it helps